hk/parser.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Human Kinetics
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+  let match;
+
+  if ((match = /^\/view\/journals\/([a-z]+)\/([0-9]+)\/([0-9]+)\/([a-z0-9-]+)\.xml$/i.exec(path)) !== null && param.content == 'pdf') {
+    // https://journals.humankinetics.com/view/journals/japa/30/5/article-p761.xml?content=pdf
+    result.rtype = 'ARTICLE';
+    result.mime = 'PDF';
+    result.title_id = match[1];
+    result.vol = match[2];
+    result.issue = match[3];
+    result.unitid = match[1] + '-' + match[2] + '-' + match[3] + '-' + match[4];
```

Comment:
For this parser , 'article-p761' is enough for unitid, use only match[4]

Reply:
This would not create a unique identifier and things might get deduplicated if one was depending on this being unique

Reply:
i don't think that this platform should have two ressource with id  'article-p761' on this route. Exept if you can found this. Do you have access to this platform to check that ?